Leanplum-SDK/Classes/Leanplum.m

```diff
@@ -886,6 +886,7 @@ + (void)startWithUserId:(NSString *)userId
         state.hasStarted = YES;
         state.startSuccessful = YES;
         NSDictionary *response = [LPResponse getLastResponse:json];
+        NSLog(@"RESPONSE: %@", response);
```

Comment:
Intentional NSLog?
I'd say we should at least add `Leanplum - ` prefix, or use LPLOG